projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/jobData.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This enum is automatically generated from the enum from  ../vdk-jupyterlab-extension/ui_job_properties.py
+ * Using https://github.com/Syndallic/py-to-ts-interfaces#example
+ */
+export enum JobProperties {
+  name = 'jobName',
```

Comment:
shouldn't the auto-generated enum be in a separate file then? are the rest of the file contents also generated or merged in a way?

Reply:
You are right. It will be better to separate it. Going to change it now